source/mainnet/smart-contracts/onboarding-guide-ethereum-developers/faq.rst

```diff
@@ -0,0 +1,371 @@
+.. _faq:
+
+====
+FAQs
+====
+
+This is a list of Frequently Asked Questions about Concordium. It is focused on
+helping developers with an Ethereum/solidity background to understand
+the Concordium blockchain and its smart contract ecosystem.
+
+Feel free to suggest additional FAQs by contacting `support@concordium.software`.
```

Comment:
Could we suggest discourse instead or in addition here?

Support email is fairly closed/one on one, but discourse is public and others can find issues there.